hack/generate-unified-csv.sh

```diff
@@ -42,29 +42,58 @@ fi
 
 hack/source-manifests.sh
 
-# Merge component-level operators into ocs CSV
-$CSV_MERGER \
-	--csv-version="$CSV_VERSION" \
-	--replaces-csv-version="$REPLACES_CSV_VERSION" \
-	--skip-range="$SKIP_RANGE" \
-	--rook-csv-filepath=$ROOK_CSV \
-	--noobaa-csv-filepath=$NOOBAA_CSV \
-	--ocs-csv-filepath=$OCS_CSV \
-	--rook-image="$ROOK_IMAGE" \
-	--ceph-image="$CEPH_IMAGE" \
-	--rook-csi-ceph-image="$ROOK_CSI_CEPH_IMAGE" \
-	--rook-csi-registrar-image="$ROOK_CSI_REGISTRAR_IMAGE" \
-	--rook-csi-resizer-image="$ROOK_CSI_RESIZER_IMAGE" \
-	--rook-csi-provisioner-image="$ROOK_CSI_PROVISIONER_IMAGE" \
-	--rook-csi-snapshotter-image="$ROOK_CSI_SNAPSHOTTER_IMAGE" \
-	--rook-csi-attacher-image="$ROOK_CSI_ATTACHER_IMAGE" \
-	--noobaa-core-image="$NOOBAA_CORE_IMAGE" \
-	--noobaa-db-image="$NOOBAA_DB_IMAGE" \
-	--ocs-image="$OCS_IMAGE" \
-	--ocs-metrics-exporter-image="$OCS_METRICS_EXPORTER_IMAGE" \
-	--ocs-must-gather-image="$OCS_MUST_GATHER_IMAGE" \
-	--crds-directory="$OUTDIR_CRDS" \
-	--manifests-directory=$BUNDLEMANIFESTS_DIR \
-	--olm-bundle-directory="$OCS_FINAL_DIR" \
-	--timestamp="$TIMESTAMP" \
-	--rook-csiaddons-image="$ROOK_CSIADDONS_IMAGE"
+if [ "$FUSION" == "true" ]; then
```

Comment:
In the if else condition I see just --ocs-csv-filepath being different, Can we do something like,
where we just set a variable like $OCS_CSV_FILEPATH accordingly if FUSION is true or false.
Then just call the CSV_MERGER tool without the if else

Reply:
Could do. But this allows us to change any arg without adding an explicit if..else per arg to change it in the future.